### PR TITLE
Modelm101 teensy2

### PR DESCRIPTION
**Pull request by wangfuco**
## Description

I added the keyboard Modelm101 teensy2 which is a variant of Modelm101 that uses teensy2.0.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/converter/modelm101_teensy2/modelm101_teensy2.c [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    writePinHigh(E6);
+  }
+  // if (usb_led & (1<<USB_LED_SCROLL_LOCK)) {
+  //   writePinLow(B5);
+  // } else {
+  //   writePinHigh(B5);
+  // }
+
+  led_set_user(usb_led);
+}
```
Comment (zvecr) [inline keyboards/converter/modelm101_teensy2/modelm101_teensy2.c:46]:
```suggestion
```

File: keyboards/converter/modelm101_teensy2/keymaps/iw0rm3r/keymap.c [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
Comment (zvecr) [inline keyboards/converter/modelm101_teensy2/keymaps/iw0rm3r/keymap.c:43]:
```suggestion
```

File: keyboards/converter/modelm101_teensy2/keymaps/iw0rm3r/config.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 iw0rm3r
```
Comment (zvecr) [inline keyboards/converter/modelm101_teensy2/keymaps/iw0rm3r/config.h:1]: This file can be deleted.

File: keyboards/converter/modelm101_teensy2/config.h [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION ROW2COL
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed (5 is default) */
+#define DEBOUNCE 5
+
+/* The Model M has no diodes */
+#define MATRIX_HAS_GHOST
+
```
Comment (zvecr) [inline keyboards/converter/modelm101_teensy2/config.h:55]:
```suggestion

/* Indicator LEDs */
#define LED_CAPS_LOCK_PIN E6
#define LED_PIN_ON_STATE 0

```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by wangfuco — COMMENTED** [review]
Thank you for the suggestions! I have applied all of them.

**Comment by fauxpark** [comment]
`.noci` can be deleted.

**Comment by wangfuco** [comment]
> `.noci` can be deleted.

It has been deleted!

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**